app/controllers/users_controller.rb

```diff
@@ -103,6 +103,7 @@ def user_params
       :is_company, :company_name, :company_info,
       :application_about, :application_motivation, :application_gender_identification, :application_age,
       :application_coding_level, :application_community_engagement, :application_language_learning_period,
+      :opted_in_newsletter, :opted_in_announcements, :opted_in_marketing_announcements, :opted_in_surveys, :opted_in_sponsorships, :opted_in_applications_open,
```

Comment:
Just a suggestion: to keep the two lists in sync, why not just pull them directly from where they are defined?
```ruby
*EmailPreferences::ATTRIBUTES
```